application/app/models/project.rb

```diff
@@ -60,6 +60,18 @@ def upload_bundles
       end
   end
 
+  def event_list
+    @event_list ||= ProjectEventList.new(project_id: id)
+  end
+
+  def all_events
```

Comment:
I would remove `all_events` and `events` from here and add these methods to the ProjectEventList class

Reply:
done